src/main/java/teammates/ui/webapi/action/GetSessionResultsAction.java

```diff
@@ -25,7 +26,7 @@ protected AuthType getMinAuthLevel() {
 
     @Override
     public void checkSpecificAccessControl() {
-        if (userInfo.isAdmin) {
+        if (userInfo != null && userInfo.isAdmin) {
```

Comment:
Just a note that this was added by me because the test failed locally after I resolved the merge conflict.